core/new-gui/src/app/workspace/component/result-panel/result-panel.component.ts

```diff
@@ -109,6 +117,31 @@ export class ResultPanelComponent {
     modalComponentInstance.currentPageSize = this.currentPageSize;
   }
 
+  /**
+   * Handler for high lighted operator.
+   * When use click on an operator, check whether it is a sink operator and display result.
+   */
+  public handleHighLightOperator(): void {
+    const highlightedOperators = this.workflowActionService.getJointGraphWrapper().getCurrentHighlightedOperatorIDs();
+
+    if (highlightedOperators.length === 1) {
+      this.showResultPanel = true;
+      this.selectedOperatorID = highlightedOperators[0];
+      const resultMap = this.executeWorkflowService.getResultMap();
+      const result: ResultObject | undefined = resultMap.get(this.selectedOperatorID);
+      if (result) {
+        this.displayResultTable(result.table);
+        if (result.chartType) {
+          this.chartType = result?.chartType;
+        } else {
+          this.chartType = undefined;
+        }
+      }
+    } else {
+      this.showResultPanel = false;
```

Comment:
Continue with the `selectedOperatorID` point above, here  `selectedOperatorID` is not changed back to `undefined`, which creates a inconsistency between local and global state.